bin/snapshot-backups/README-DR.md

```diff
@@ -12,33 +115,22 @@
         For example:
         USAGOV-784-defacement-recovery.dev.4250.process_test_001
 
-### Static site backup
-        export echo=echo
+### Manually Tagged Static site backup
+
+        echo=echo   ### Do NOT export echo. Ask me how I know this (tm).
+        dryrun='--dryrun'
```

Comment:
Just to be picky, let's either remove "dryrun='--dryrun'" from these last couple of examples, or actually use ${dryrun} in the examples.

Reply:
good call - added.